haproxy/dockerdir/bin/check_pxc.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o xtrace
+
+PXC_SERVER_IP=$3
+PXC_SERVER_PORT=$4
+MONITOR_USER='monitor'
+TIMEOUT=10
+MONITOR_PASSWORD="$PATH"
+
+MYSQL_CMDLINE="/usr/bin/timeout $TIMEOUT /usr/bin/mysql -nNE -u$MONITOR_USER"
+
+PXC_NODE_STATUS=($(MYSQL_PWD="${MONITOR_PASSWORD}" $MYSQL_CMDLINE -h $PXC_SERVER_IP -P $PXC_SERVER_PORT \
+        -e "SHOW STATUS LIKE 'wsrep_local_state';SHOW VARIABLES LIKE 'pxc_maint_mode';" \
```

Comment:
should we also check `wsrep_cluster_status`? if the member is `Primary`

Reply:
fixed